universal_robot/cd273dd/cd273dd.bug

```diff
@@ -61,4 +61,4 @@ fault-codes:
   - BDO:DEPENDENCY
 failure-codes:
   - SOFTWARE:CRASHING
-  - SYSTEM:LOSS-OF-CONTROL
+  - SYSTEM:UNRESPONSIVE
```

Comment:
`LOSS-OF-FUNCTIONALITY` might also apply here. MoveIt is not used only for user-provided commands and trajectories, I think.

Reply:
I am not sure what you mean by "user provided commands and trajectories"

Reply:
From the meeting: let's go with Loss of functionality and Unresponsive.